src/embed.js

```diff
@@ -59,7 +59,7 @@ function load(url, arg, prop, el, callback) {
 function embed(el, spec, opt, callback) {
   var cb = callback || function(){}, source,
   renderer = (opt && opt.renderer) || 'canvas',
-  actions  = (opt && opt.actions) || {}, mode;
```

Comment:
Yeah I was too hurried. 
BTW, Default is `true`.
I'll commit more codes to fix this!